src/scimlfunctions.jl

```diff
@@ -2043,44 +2039,42 @@ OptimizationFunction{iip}(f, adtype::AbstractADType = NoAD();
   This is defined as an array of matrices, where `hess[i]` is the Hessian w.r.t. the `i`th output.
   For example, if the Hessian is sparse, then `hess` is a `Vector{SparseMatrixCSC}`.
   The default is `nothing`, which means a dense constraint Hessian.
+
+When [Symbolic Problem Building with ModelingToolkit](@ref) interface is used the following arguments are also relevant:
+
 - `syms`: the symbol names for the elements of the equation. This should match `u0` in size. For
   example, if `u = [0.0,1.0]` and `syms = [:x, :y]`, this will apply a canonical naming to the
   values, allowing `sol[:x]` in the solution and automatically naming values in plots.
 - `paramsyms`: the symbol names for the parameters of the equation. This should match `p` in
   size. For example, if `p = [0.0, 1.0]` and `paramsyms = [:a, :b]`, this will apply a canonical
   naming to the values, allowing `sol[:a]` in the solution.
-- `hess_colorvec`: a color vector according to the SparseDiffTools.jl definition for the sparsity
```

Comment:
?

Reply:
I have added the fields back but removing it from docstrings seems reasonable until there is a release with support for it?

Reply:
These pieces are still correct if they are unused though. It's not like it leads to any correctness, it's just lack of optimization for them to not be used.

Reply:
But as a user, it would be confusing to see documented fields that are not used anywhere? It will be trivial to add them back as soon as their use gets implemented

Reply:
> It will be trivial to add them back as soon as their use gets implemented

We have a policy of not going this direction because it never happens as planned. Docs of interfaces stay complete. If a package isn't matching the interface, the package should change.

> But as a user, it would be confusing to see documented fields that are not used anywhere?

We've never had issues with that in the past. Not useful features are generally just ignored until they are useful. And this one is deep in the weeds.

Reply:
Added it back